pkg/costmodel/cluster.go

```diff
@@ -263,6 +269,10 @@ func ClusterDisks(client prometheus.Client, cp models.Provider, start, end time.
 			continue
 		}
 
+		if !env.GetAssetIncludeLocalDiskCost() && strings.HasPrefix(volumeName, SIG_STORAGE_LOCAL_PROVISIONER_PREFIX) {
```

Comment:
I am confused what this has to do with this env variable. The env variable is about including node attached disk where as it appears that these disks are PVs

Reply:
PVs created from sig-storage-local-provisioner are volumes/disk carved for local storage disk of the node. [Sig-storage-local-static-provisioner ](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner) gives impression of a pv attached from a local disk of a node. 

So even though they are PVs, they're actually encompassed in the node cost itself, similar to a node-attached disk. 

@thomasvn  wanted to skip the cost for this disks leveraging the bool added in [PR](https://github.com/opencost/opencost/pull/2441), correct me if im wrong here.

Reply:
@avrodrigues5 Yes this is correct! They will show up as k8s PVs, but under the hood sig-storage-local-provisioner is creating those PVs using the node's local disk.

Reply:
Okay but if they are showing up as PV's in the k8s api, then they are showing up as PV's in the prom metrics. Which means it has nothing to do with that env variable

Reply:
@Sean-Holcomb I'm unsure what you want me to do with respect to the environment variable? Please let me know what is expected.

Great suggestion on filtering diskmap at the end before returning. I have made changes and updated the test around it too.